src/instructlab/cli/data/ingest.py

```diff
@@ -0,0 +1,128 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Standard
+import logging
+
+# Third Party
+import click
+
+# First Party
+from instructlab import clickext
+from instructlab.data.ingest_docs import ingest_docs
+from instructlab.data.taxonomy_utils import lookup_processed_documents_folder
+from instructlab.defaults import DEFAULTS
+from instructlab.rag.rag_configuration import (  # type: ignore
+    document_store_configuration,
+    embedder_configuration,
+)
+
+logger = logging.getLogger(__name__)
+
+
+# TODO: fill-in help fields
+@click.command()
+@click.option(
+    "--document-store-type",
+    default="milvuslite",
+    envvar="ILAB_DOCUMENT_STORE_TYPE",
+    type=click.STRING,
+    help="The document store type, one of: `milvuslite`, `milvus`.",
+)
+@click.option(
+    "--document-store-uri",
+    default="embeddings.db",
+    envvar="ILAB_DOCUMENT_STORE_URI",
+    type=click.STRING,
+    help="The document store URI",
+)
+@click.option(
+    "--document-store-collection-name",
+    default="IlabEmbeddings",
+    envvar="ILAB_DOCUMENT_STORE_COLLECTION_NAME",
+    type=click.STRING,
+    help="The document store collection name",
+)
+@click.option(
+    "--model-dir",
+    default=lambda: DEFAULTS.MODELS_DIR,
+    envvar="ILAB_MODEL_DIR",
+    show_default="The default system model location store, located in the data directory.",
+    help="Base directories where models are stored.",
+)
+@click.option(
+    "--embedding-model",
+    "embedding_model_name",
+    default="sentence-transformers/all-minilm-l6-v2",
+    envvar="ILAB_EMBEDDING_MODEL_NAME",
+    type=click.STRING,
+    help="The embedding model name",
+)
+@click.option(
+    "--output-dir",
+    envvar="ILAB_OUTPUT_DIR",
+    help="Directory where generated datasets are stored.",
+)
+@click.argument(
```

Comment:
we tend, for now, to stay away from using click arguments. Can we use `.option` for everything here?

Reply:
sure, I will also update the dev-docs PR accordingly